src/mission_start.cpp

```diff
@@ -30,7 +30,7 @@ point random_house_in_city( const city_reference &cref )
     for( int x = startx; x <= endx; x++ ) {
         for( int y = starty; y <= endy; y++ ) {
             if( overmap_buffer.check_ot_type( "house", x, y, 0 ) ) {
-                valid.push_back( point( x, y ) );
+                valid.push_back( tripoint( x, y, 0 ) );
```

Comment:
`city_reference` has now a tripoint, why not use its z-component here instead of the fixed 0?